packages/flutter/test/widgets/page_view_test.dart

```diff
@@ -433,6 +461,7 @@ void main() {
         child: PageView(
           pageSnapping: pageSnapping,
           onPageChanged: log.add,
+          pageDidChanged: log.add,
```

Comment:
There should probably be a different log variable so we can differentiate in the test which callback fired.